scripts/server

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
-set -eu
+set -e
+
+if [[ -n "${GIT_COMMIT}" ]];
```

Comment:
I'm seeing this on DistrictBuilder core:
```
vagrant@vagrant-ubuntu-trusty-64:/vagrant$ ./scripts/server
fatal: Not a git repository (or any of the parent directories): .git
```

Reply:
It looks like we exclude `.git` in the `Vagrantfile`

Reply:
Ah, I forgot I've been running the app outside of the VM. Do you have a preference between "set default `GIT_COMMIT` to `latest`"  and "remove .git/ from rsync excludes"?